src/speaker.c

```diff
@@ -0,0 +1,68 @@
+#include "speaker.h"
+
+uint8_t BASE_NOTE = 9;
+uint8_t BASE_OCTAVE = 4;
+float BASE_FREQUENCY = 440.;
+uint8_t POSITIONS[8] = {9, 11, 0, 2, 4, 5, 7};
```

Comment:
Explain these numbers

Reply:
Fixed in 9c06a76717413974d25939122a23e8d7601f7cd4.

Reply:
I like ti <3